### PR TITLE
Mention publisher rewards

### LINKED ISSUE #462: Mention higher publisher rewards

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #462.

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -217,8 +217,13 @@
 
 <Router>
   <BannerMessage
-    title="This software is a Beta release."
-    message={`The website is under construction, use at your own discretion.`}
+    title={`This software is a Beta release.`}
+    message={`The website is under construction, use at your own discretion`}
```
Comment (idiom-bytes) [inline src/App.svelte:221]: Can we disable the first banner? I don't think we need it anymore and should only have 1 of them.

The publisher reward message below should use the golden styling.

File: src/components/claim/ClaimRewardsVeDF.svelte [hunk 2]
```diff
@@ -147,7 +147,7 @@
         : ""
     }`}
     apyTooltip={descriptions.default.tooltip_rewards_apy_active}
-    description="Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>."
+    description="Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>.<br/>Get <strong>2X Rewards by publishing</strong> your own datasets and allocating to them."
```
Comment (idiom-bytes) [inline src/components/claim/ClaimRewardsVeDF.svelte:150]: Don't use a <br/>, the styling/formatting looks bad.

Can you also update the Passive Rewards card to not have a <br/>? It doesn't look good at the moment.

Comment (EnzoVezzaro) [reply src/components/claim/ClaimRewardsVeDF.svelte:150]: can you also add some spaces between paragraphs. It creates some space to breath. 

Screenshot 2023-02-21 at 11 36 51

File: src/App.svelte (hunk 1 above)
Comment (EnzoVezzaro) [reply src/App.svelte:221]: > although we are still in the Beta phase.

I would still keep it for now, We can just change the message to something like this maybe: "This software is a Beta release. Use at your own discretion".

Screenshot 2023-02-21 at 11 41 40

File: src/components/claim/ClaimRewardsVeDF.svelte (hunk 2 above)
Comment (KatunaNorbert) [reply src/components/claim/ClaimRewardsVeDF.svelte:150]: @EnzoVezzaro sure, looks better

File: src/App.svelte (hunk 1 above)
Comment (idiom-bytes) [reply src/App.svelte:221]: Line 1 => Data Farming [is in Beta](https://blog.oceanprotocol.com/ocean-data-farming-is-launching-f633532d831c). Use at your own discrtion.
Line 2 => Publish your datasets trough our [marketplace](https://market.oceanprotocol.com/publish/1) to get [2X Rewards](https://blog.oceanprotocol.com/data-farming-publisher-rewards-f2639525e508) for publishers.

File: src/components/common/BannerMessage.svelte [hunk 3]
```diff
@@ -30,12 +34,16 @@
...
+  }
   span {
     font-weight: bold;
     margin-right: calc(var(--spacer) / 12);
   }
   p {
-    margin: 0;
+    margin-bottom: 0;
     font-weight: 500;
     font-size: var(--font-size-mini);
```
Comment (EnzoVezzaro) [inline src/components/common/BannerMessage.svelte:48]: I'd keep the font size of the banner the same as the market's: `font-size: var(--font-size-small);`

**Review by EnzoVezzaro — APPROVED** [review]
LGTM. I just commented about the font-size. After solving the conflicts with main and deciding what to do with the font-size, we can ship this.

**Review by EnzoVezzaro — APPROVED** [review]
great job 👍🏽

**Pull request merged**